lib/Authentication/AuthTokens/X509AuthenticationToken.php

```diff
@@ -26,7 +26,7 @@ public function __construct() {
         //$this->logger->pushHandler(new StreamHandler(__DIR__.'/../../../gocdb.log', Logger::DEBUG));
 
         $this->initialDN = $this->getDN();
-        $this->userDetails = array('AuthenticationRealm' => array('IGTF'));
+        $this->userDetails = array('AuthenticationRealm' => array('IGTF X509 Cert'));
```

Comment:
Is there anything in the program that enforces the X509 'flavour' to be IGTF only? What set of certificate issuers is accepted is set by the Apache/server configuration I think. Would plain 'X509' be more understandable/general?

Reply:
I think I may have originally suggested `IGTF X509 Cert`, but you are right @ineilson, there is nothing in the codebase that enforces the 'flavour' that's all handled by apache.

`X.509` _is_ more future proof (though I can't see a requirement for non IGTF certificate based access appearing anytime soon).

Reply:
This was done.